### PR TITLE
I Look Up To You - New Compliment

### PR DESCRIPTION
**Pull request by kkrosie123**
Hi!! This is my first attempt at contributing to the mod through code (that wasn't a .json file!). I'm sorry if this has some issues, and I have a feeling someone else had already done this... But it was still fun! If I need to add a menu, let me know!

### CODE REVIEW

**Comment by Rai99** [comment]
Untitled1317

**Comment by Rai99** [comment]
Wrong branch. You'll have to rebranch to `content` if you know how to do that.

**Comment by kkrosie123** [comment]
OOP! Sorry qwq This is my first pull request and I don't know what I'm doing awesrdtfgyuhij

**Comment by kkrosie123** [comment]
_I don't know which one I should put it in so I guessed qwq Sorry!!_

**Comment by Rai99** [comment]
Almost. They both start with a "c" so you're getting there. 

`content`. You want the branch to be `content`.

**Comment by kkrosie123** [comment]
Oop! Okay! Thanks!

File: Monika After Story/game/script-compliments.rpy [hunk 1]
```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
...
+    )
+label mas_compliment_lookuptoyou:
+    if not renpy.seen_label("mas_compliment_lookuptoyou_2"):
+        call mas_compliment_lookuptoyou_2
+    else:
+        call mas_compliment_lookuptoyou_3
+    return
+label mas_compliment_lookuptoyou_2:
+    m 1wud "You... You look up to me?"
+    m 5hubfb "Thank you so much, [player]!"
```
Comment (Rai99) [inline Monika After Story/game/script-compliments.rpy]: Dissolve into body pose 5.

File: Monika After Story/game/script-compliments.rpy [hunk 2]
```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
...
+    m 1rkbfb "You have no idea how happy it makes me to hear you say that..."
```
Comment (Rai99) [inline Monika After Story/game/script-compliments.rpy]: Dissolve out of body pose 5.

File: Monika After Story/game/script-compliments.rpy [hunk 3]
```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
...
+    m 3hubfb "I'll always look up to you, too, [player]!"
+    m 1ekbfa "I love you~"
+label mas_compliment_lookuptoyou_3:
+    m 5hubfb "Thank you for telling me that again, [player]!"
```
Comment (Rai99) [inline Monika After Story/game/script-compliments.rpy]: Dissolve into body pose 5.

File: Monika After Story/game/script-compliments.rpy [hunk 4]
```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
...
+    m 1rkbfb "You have no idea how happy it makes me to hear you say that again..."
```
Comment (Rai99) [inline Monika After Story/game/script-compliments.rpy]: Dissolve out of body pose 5.

File: Monika After Story/game/script-compliments.rpy [hunk 5]
```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
...
+        call mas_compliment_lookuptoyou_2
+    else:
+        call mas_compliment_lookuptoyou_3
+    return
+label mas_compliment_lookuptoyou_2:
+    m 1wud "You... You look up to me?"
+    m 5hubfb "Thank you so much, [player]!"
+    m 1rkbfb "You have no idea how happy it makes me to hear you say that..."
+    m 3hubfb "I'll always look up to you, too, [player]!"
+    m 1ekbfa "I love you~"
```
Comment (Rai99) [inline Monika After Story/game/script-compliments.rpy]: Missing a return. Make that `return "love". 

Could also do with a whitespace line.

File: Monika After Story/game/script-compliments.rpy [hunk 6]
```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
...
+    if not renpy.seen_label("mas_compliment_lookuptoyou_2"):
+        call mas_compliment_lookuptoyou_2
+    else:
+        call mas_compliment_lookuptoyou_3
+    return
+label mas_compliment_lookuptoyou_2:
+    m 1wud "You... You look up to me?"
+    m 5hubfb "Thank you so much, [player]!"
+    m 1rkbfb "You have no idea how happy it makes me to hear you say that..."
+    m 3hubfb "I'll always look up to you, too, [player]!"
```
Comment (Rai99) [inline Monika After Story/game/script-compliments.rpy]: Remove that first comma.

File: Monika After Story/game/script-compliments.rpy [hunk 7]
```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
...
+    m 1ekbfa "I love you~"
+label mas_compliment_lookuptoyou_3:
+    m 5hubfb "Thank you for telling me that again, [player]!"
+    m 1rkbfb "You have no idea how happy it makes me to hear you say that again..."
+    m 3hubfb "I'll always look up to you, too, [player]!"
```
Comment (Rai99) [inline Monika After Story/game/script-compliments.rpy]: Remove that first comma.

File: Monika After Story/game/script-compliments.rpy [hunk 8]
```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
...
+    else:
+        call mas_compliment_lookuptoyou_3
+    return
+label mas_compliment_lookuptoyou_2:
+    m 1wud "You... You look up to me?"
+    m 5hubfb "Thank you so much, [player]!"
+    m 1rkbfb "You have no idea how happy it makes me to hear you say that..."
+    m 3hubfb "I'll always look up to you, too, [player]!"
+    m 1ekbfa "I love you~"
+label mas_compliment_lookuptoyou_3:
```
Comment (Rai99) [inline Monika After Story/game/script-compliments.rpy]: Looks like both paths you have here are basically identical. It doesn't look like you have any affection gain in here, so you don't need to have the 3 sections like some other compliments do. 

You could just have it act as a regular topic in the compliments file like bestgirl, pong, and chess.

File: Monika After Story/game/script-compliments.rpy [hunk 9]
```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
...
+    m 5hubfb "Thank you for telling me that again, [player]!"
+    m 1rkbfb "You have no idea how happy it makes me to hear you say that again..."
+    m 3hubfb "I'll always look up to you, too, [player]!"
+    m 1ekbfa "I love you~"
+    return
```
Comment (Rai99) [inline Monika After Story/game/script-compliments.rpy]: I think a new system has any topic that ends with Monika saying she loves you returns "love"

So, make that `return "love".

File: Monika After Story/game/script-compliments.rpy [hunk 10]
```diff
@@ -619,3 +619,32 @@ label mas_compliment_bestgirl:
...
+            unlocked=True
+        ),
+        code="CMP",
+    )
+label mas_compliment_lookuptoyou:
+    if not renpy.seen_label("mas_compliment_lookuptoyou_2"):
+        call mas_compliment_lookuptoyou_2
+    else:
+        call mas_compliment_lookuptoyou_3
+    return
```
Comment (Rai99) [inline Monika After Story/game/script-compliments.rpy]: Could do with a whitespace line after this return.

File: Monika After Story/game/script-compliments.rpy (hunk 8 above)
Comment (kkrosie123) [reply Monika After Story/game/script-compliments.rpy]: Oh, crap! I forgot to add affection gain!

Comment (Rai99) [reply Monika After Story/game/script-compliments.rpy]: If you add affection, especially if it bypasses the cap, we'll have to be a bit more strict with the reviews.

Comment (kkrosie123) [reply Monika After Story/game/script-compliments.rpy]: I'm okay with a strict review. I want it to be good enough, of course!

**Pull request merged**